gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -101,6 +101,10 @@ class {{ service.name }}RestTransport({{ service.name }}Transport):
                     self._host,
                     credentials=self._credentials,
                     scopes=self.AUTH_SCOPES,
+                    options=[
```

Comment:
I added the options wherever a channel was created, but am not 100% sure if it makes sense here. @yon-mg could you take a look?